packages/devtools-launchpad/src/test/utils/debuggee.js

```diff
@@ -1,120 +0,0 @@
-function Debuggee() {
```

Comment:
dropped debuggee so that we can move it to the debugger. Eventually, we'll pull this stuff out into its own `mocha-redux` repo